common/djangoapps/student/views/management.py

```diff
@@ -741,6 +742,13 @@ def create_account_with_params(request, params):
     else:
         compose_and_send_activation_email(user, profile, registration)
 
+    if settings.FEATURES.get('AUTO_SOFTWARE_SECURE_PHOTO_VERIFICATION'):
+        SoftwareSecurePhotoVerification.objects.create(
+            status="approved",
+            user_id=user.id,
+            submitted_at=datetime.datetime.now(UTC),
+            photo_id_key=uuid.uuid4())
```

Comment:
Please, move the trailing bracket to the next line.

Reply:
Will be changed in next commit